### PR TITLE
Add Clear Linux Project documentation

### PR DESCRIPTION
**Pull request by bryteise**
Create documentation to go with https://github.com/docker-library/official-images/pull/1721.

### CODE REVIEW

File: clearlinux/README-short.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Official docker build of Clear Linux OS for Intel Architecture
```
Comment (tianon) [inline clearlinux/README-short.txt]: The image itself isn't Intel-only, is it?

Comment (tianon) [inline clearlinux/README-short.txt]: (for context, this is the blurb that'll show up in `docker search` and at the top of the Hub description page)

**Comment by tianon** [comment]
I think this is a good start, but I'd love it if we could add some examples of how to use the image a little bit (for example, a simple `Dockerfile` showing how to `FROM clearlinux:base` and install a few packages or something similarly trivial).

**Comment by tianon** [comment]
Ok, managed to find https://clearlinux.org/documentation/bundles_overview.html, which IMO would be good to link to. :+1:

Here's essentially the kind of simple `Dockerfile` I'm thinking would be worthwhile to add:
```dockerfile
FROM clearlinux:base
RUN swupd bundle-add editors
```
(which I've just verified works well)

File: clearlinux/README-short.txt (hunk 1 above)
Comment (bryteise) [inline clearlinux/README-short.txt]: It's the official name we use (either OS or Project, in this case OS made more sense to me).

**Comment by bryteise** [comment]
I'll start working on flushing out the documentation then, and thanks for the test!

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**